src/js/components/DataFilter/DataFilter.js

```diff
@@ -123,6 +123,7 @@ export const DataFilter = ({
     if (range) {
       content = (
         <RangeSelector
+          aria-label={property}
```

Comment:
Shouldn't this do what the FormField would do for label and try to use a label defined in the properties? In many real world cases that property name isn't all that friendly/usable.
```jsx
label={properties?.[property]?.label || property}
```

Reply:
good point yes the `property` isnt always the best